src/vcConvert.cpp

```diff
@@ -1,6 +1,8 @@
 #include "vcConvert.h"
 
 #include "imgui.h"
+#include "imgui_internal.h"
```

Comment:
No comment for what requires this?

Reply:
`SeparatorEx` in case people look at this in future